sdk/core/core-auth/src/tokenCredential.ts

```diff
@@ -79,8 +105,24 @@ export interface AccessToken {
    * The timestamp when the access token should be refreshed, in milliseconds, UNIX epoch time.
    */
   refreshAfterTimestamp?: number;
+
+  /** Type of token - Bearer or PoP */
+  tokenType: string;
```

Comment:
Do we want some sort of enum here?

Reply:
Or perhaps `tokenType: string = "Bearer"`

Reply:
though I guess we can't actually make it required without it being a break since it's an input type

Reply:
yeah, this should be OK to make a union, right -- don't think it would be breaking to add other types later if needed?

```js
export type TokenType = "Bearer" | "PoP";

export interface AccessToken {
  // ...

  /**
   * Type of token. Defaults to "Bearer" if unset.
   */
  tokenType?: TokenType;
}
```

Reply:
This was discussed in the archboard meeting - we specifically discussed about leaving this as a string. So that we can use it to form the authorization header.